site/src/components/carousel.tsx

```diff
@@ -102,6 +104,7 @@ export const Carousel = <T,>({
 
   return (
     <Box
+      data-testid={props["data-testid"]}
```

Comment:
Do we need this to be a variable? A hardcoded `data-testid` value could work too in this case. If we have multiple block carousels, we can select them via `...parent... >> [data-testid="block-slider (or some other name)"]`.